mkt/search/views.py

```diff
@@ -72,13 +77,23 @@ def get_serializer_context(self):
         return context
 
     def get_queryset(self):
+        # Check if we are filtering by a doc_type (e.g., apps, sites).
+        # Default to all content types.
+        app_mapping_name = WebappIndexer.get_mapping_type_name()
+        site_mapping_name = WebsiteIndexer.get_mapping_type_name()
+        doc_type_list = [app_mapping_name, site_mapping_name]
+        doc_type = self.request.GET.get('doc_type', 'all')
+        if doc_type == 'app':
+            doc_type_list = [app_mapping_name]
+        elif doc_type == 'site':
+            doc_type_list = [site_mapping_name]
+
         excluded_fields = list(set(WebappIndexer.hidden_fields +
                                    WebsiteIndexer.hidden_fields))
-        return (Search(
-            using=BaseIndexer.get_es(),
-            index=[WebappIndexer.get_index(), WebsiteIndexer.get_index()],
-            doc_type=[WebappIndexer.get_mapping_type_name(),
-                      WebsiteIndexer.get_mapping_type_name()])
+        return (Search(using=BaseIndexer.get_es(),
+                       index=[WebappIndexer.get_index(),
+                              WebsiteIndexer.get_index()],
```

Comment:
Ah, yeah, I suggested to only select the `doc_type`, but it's probably best to select the index too.